package.json

```diff
@@ -46,5 +47,8 @@
     "ts-loader": "^8.0.1",
     "tsdx": "^0.13.2",
     "typescript": "^3.9.6"
+  },
+  "dependencies": {
+    "theme-ui": "^0.3.1"
```

Comment:
this must be a peer dependency, and in order not to break your application, you also put it as a dev dependency